tests/test_route53.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for the AWS Route53 helper module."""
+
+from app.route53 import _derive_fully_specified_domain
+from app.route53 import _lexicographic_order_domain
+
+
+def test_fsd():
+    assert _derive_fully_specified_domain('lsst.io.') \
+        == 'lsst.io.'
+
+    assert _derive_fully_specified_domain('lsst.io') \
+        == 'lsst.io.'
+
+    assert _derive_fully_specified_domain('pipelines.lsst.io') \
+        == 'lsst.io.'
+
+    assert _derive_fully_specified_domain('//pipelines.lsst.io') \
```

Comment:
See comment above about this not being a valid URL.  What is the use case for this format?